Packs/RegexReplace/Scripts/RegexReplace/RegexReplace.yml

```diff
@@ -0,0 +1,51 @@
+args:
+- description: Text to match against
+  name: value
+  required: true
+- description: Regex pattern to search
+  name: regex
+  required: true
+- description: Template string to format patterns matched with regex
+  name: output_format
+  required: true
+- auto: PREDEFINED
+  default: true
+  defaultValue: "false"
+  description: True if the pattern matching take places in case insensitive, otherwise
+    False.
+  name: ignore_case
+  predefined:
+  - "true"
+  - "false"
+comment: |-
+  Format patterns matched with regex.
+
+  Example 1:
+   value: user=john
+   regex: user=(.*)
+   output_format: name=\1
+   -> output value: name=john
+
+  Example 2:
+   value: xxx=yyy
+   regex: user=(.*)
+   output_format: name=\1
+   -> output value: xxx=yyy
```

Comment:
Not sure I understand this example - if the regex does not find any match, this script is returning the original value? If so, it is worth mentioning in the description.

Reply:
Yes, the original value is returned for the unmatched pattern.